portfolio/src/main/webapp/scripts/base.js

```diff
@@ -1,4 +1,20 @@
 $("document").ready(function() {
-  $("header").load('content/header.html');
+  let templatePromise = loadTemplate('content/header.html');
+  let objPromise = fetch('/auth').then(promiseResponse => promiseResponse.json());
+  Promise.all([templatePromise, objPromise]).then((values) => {
+    let template = values[0];
+    let obj = values[1];
```

Comment:
some more descriptive variable names would have been helpful here. It should describe that its login status information.